### PR TITLE
Create 2015-06-17-shapes-clusters

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp how is this?  I don't know what to put for the name.  should we make the code nice/ cleaner?

### CODE REVIEW

File: _posts/python/shapes/2015-06-17-shapes-clusters [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+name: Styling Clusters with Shapes
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-clusters]: maybe: `Highlighting Clusters of Scatter Points with Circle Shapes`?

**Comment by chriddyp** [comment]
love it! :dancer:

File: _posts/python/shapes/2015-06-17-shapes-clusters [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+)
+
+
+layout = {
+    'shapes':[
+    {
+        'type': 'circle',
+        'xref': 'x',
+        'yref': 'y',
+        'x0': 0.75,
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-clusters]: we *could* do something like `min(x0)` here

File: _posts/python/shapes/2015-06-17-shapes-clusters [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        'y0': 0.75,
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-clusters]: and `min(y0)`

File: _posts/python/shapes/2015-06-17-shapes-clusters [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        'x1': 3.5,
```
Comment (chriddyp) [inline _posts/python/shapes/2015-06-17-shapes-clusters]: `max(x0)` etc

**Comment by aneda** [comment]
@chriddyp it's ready!

**Pull request merged**